build: upgrade `webdriver-manager` to v12.1.8

gkalpak opened the pull request:
ChromeDriver now supports Apple Silicon ARM processors. `webdriver-manager` versions 12.1.7 and earlier will, however, incorrectly download the arm64 ChromeDriver regardless of the system's architecture. This results in failure to run Protractor tests on macOS with the error: `spawn Unknown system error -86`

This commit fixes the problem by upgrading `webdriver-manager` to version 12.1.8, which includes a fix.
See also https://stackoverflow.com/questions/65618558.

mary-poppins commented:
You can preview 5fdedc1 at https://pr40756-5fdedc1.ngbuilds.io/.

petebacondarwin commented on integration/ivy-i18n/package.json:
```diff
@@ -31,28 +31,27 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "file:/private/var/tmp/_bazel_pete/f9acfb7f019473a10a34c8c30adc55ea/execroot/angular/bazel-out/darwin-fastbuild/bin/integration/ivy-i18n_test.debug.sh.runfiles/angular/packages/animations/npm_package_archive.tar.gz",
```
Who is this `bazel_pete`? Sounds invidious.

gkalpak replied:
Totally 😛

mary-poppins commented:
You can preview 9ae6d92 at https://pr40756-9ae6d92.ngbuilds.io/.

mary-poppins commented:
You can preview d9e59c2 at https://pr40756-d9e59c2.ngbuilds.io/.

josephperrott reviewed (approved):
LGTM

josephperrott commented:
@gkalpak  Looks like this doesn't cleanly merge into the patch branch, likely because of the lock file I assume. Can you create a patch only version of the PR?

gkalpak committed:
build: ensure integration projects use root `webdriver-manager`

Ensure that all integration projects use `webdriver-manager` from the
root `node_modules/`. See acfd0edd38d955a8559aeec3d97c0f9a23525079 for
more details.

gkalpak committed:
build: use correct path for local dependencies in `ivy-i18n` integration project

Previously, the paths to some local dependencies in the `ivy-i18n`
integration project were pointing to non-generic bazel files that would
normally not exist on a user's machine.

This commit changes these paths to relative paths pointing to build
artifacts that are guaranteed to exist (once the necessary build
commands have been executed).

gkalpak committed:
build: upgrade `webdriver-manager` to v12.1.8

ChromeDriver now supports Apple Silicon ARM processors.
`webdriver-manager` versions 12.1.7 and earlier will, however,
incorrectly download the arm64 ChromeDriver regardless of the
system's architecture. This results in failure to run Protractor tests
on macOS with the error: `spawn Unknown system error -86`

This commit fixes the problem by upgrading `webdriver-manager` to
version 12.1.8, which includes a fix.
See also https://stackoverflow.com/questions/65618558.

gkalpak commented:
@josephperrott, I've created #40885 for the patch branch.
(I have also rebased this on master and labeled as `target: minor`.)

mary-poppins commented:
You can preview 4a0302f at https://pr40756-4a0302f.ngbuilds.io/.
